kernel/kernel-api/src/main/java/io/delta/kernel/TransactionBuilder.java

```diff
@@ -34,11 +34,27 @@
 @Evolving
 public interface TransactionBuilder {
   /**
-   * Set the schema of the table when creating a new table.
+   * Set the schema of the table. If setting the schema on an existing table for a schema evolution,
+   * then column mapping must be enabled. The possible schema evolutions supported include column
```

Comment:
Do we have tests that using public API requires column mapping to be enabled? I see the `@throws io.delta.kernel.exceptions.KernelException in case column mapping is not enabled` but I didn't see a test?

Perhaps I just missed it! Let me know 😄

Reply:
I wrote a test on this I'm pretty sure, let me get a link